src/app/mail/mail-sidebar/mail-sidebar.component.ts

```diff
@@ -33,12 +33,13 @@ export class MailSidebarComponent implements OnInit, OnDestroy {
 
   LIMIT = 3;
   EMAIL_LIMIT = 20;
-
+  PAGE = 1;
```

Comment:
This variable is unused, remove it, All of this page doesn't have any useful change, remove all changes from this page.